plugins/modules/sda_fabric_devices_workflow_manager.py

```diff
@@ -1328,6 +1328,73 @@ def get_fabric_site_id_from_name(self, site_id, site_name):
 
         return fabric_site_id
 
+    def get_fabric_zone_id_from_name(self, zone_id, site_name):
```

Comment:
This API is confusing. You are passing zone_id but getting zone_id? 
Can't we say as site_id as we pass site_id from the the caller?